servicemesh-install/topics/tutorial-threescale-adapter.adoc

```diff
@@ -0,0 +1,147 @@
+[[threescale-adapter-tutorial]]
+= 3scale Istio Adapter tutorial
+The 3scale Istio Adapter allows you to label a service running within the {ProductName} and integrate that service with the 3Scale API Management solution.
+
+Prerequisites:
+
+* Istio 1.1
```

Comment:
This should reference Service Mesh 0.7.0 or above rather than specifically mentioning the upstream Istio version

Reply:
Fixed.